Crash if argv is empty

```
% heartbleed
panic: runtime error: index out of range

goroutine 1 [running]:
runtime.panic(0x5c56c0, 0x816c57)
        /usr/lib/go/src/pkg/runtime/panic.c:266 +0xb6
main.main()
        .../heartbleed/src/src/github.com/FiloSottile/Heartbleed/bleed.go:10 +0x5b5
```

This needs a nicer error message :-)

--- a/bleed.go
+++ b/bleed.go
@@ -2,13 +2,33 @@ package main
 
 import (
 	bleed "github.com/FiloSottile/Heartbleed/bleed"
+	"fmt"
 	"log"
 	"net/url"
 	"os"
 )
 
+var usageMessage = `This is a tool for detecting OpenSSL Heartbleed vulnerability (CVE-2014-0160).
+
+Usage:
+
+	%s server_name(:port)
+	
+	The default port is 443 (HTTPS).
+`
+
+func usage(progname string) {
+	fmt.Fprintf(os.Stderr, usageMessage, progname)
+	os.Exit(2)
+}
+
 func main() {
-	host := os.Args[1]
+	args := os.Args
+	if len(args) < 2 {
+		usage(args[0])
+	}
+
+	host := args[1]
 	u, err := url.Parse(host)
 	if err == nil && u.Host != "" {
 		host = u.Host

